### PR TITLE
Add omero-guides to training page

### PR DESCRIPTION
**Pull request by pwalczysko**
See https://github.com/ome/www.openmicroscopy.org/issues/325 for motivation.

cc @jburel @mtbc @will-moore

### CODE REVIEW

File: training/index.html [hunk 1]
```diff
@@ -22,6 +22,9 @@ <h1>OME Training</h1>
                 materials we develop are freely available along with the image data and
                 setup procedures used.
             </p>
+            <p>
+                The content of the wokrshops is usually derived from materials in <a href="https://omero-guides.readthedocs.io/en/latest">omero-guides</a>. New topics of workshops can also be suggested or discussed. <a href="https://omero-guides.readthedocs.io/en/latest">omero-guides</a> can also be used as a starting user manual and introduction into OMERO concepts and capabilities.
```
Comment (mtbc) [inline training/index.html]:
```suggestion
                The content of the wokrshops is usually derived from materials in <a href="https://omero-guides.readthedocs.io/en/latest">omero-guides</a>. New topics of workshops can also be suggested or discussed. <a href="https://omero-guides.readthedocs.io/en/latest">omero-guides</a> can also be used as a starting user manual and introduction to OMERO concepts and capabilities.
```

**Comment by pwalczysko** [comment]
I guess this will be staged at https://snoopycrimecop.github.io/www.openmicroscopy.org when I find the job which deploys it.

**Comment by pwalczysko** [comment]
attempting to stage using https://ci.openmicroscopy.org/view/Mgmt/job/WWW-merge/

**Comment by pwalczysko** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/training/

**Comment by jburel** [comment]
I think we should have a second button next to ``Browse`` under the ``Material`` box e.g. ``Guides``

**Comment by jburel** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/training/

**Pull request merged**